sql/core/src/main/scala/org/apache/spark/sql/jdbc/PostgresDialect.scala

```diff
@@ -47,7 +47,10 @@ private case class PostgresDialect()
   private val supportedAggregateFunctions = Set("MAX", "MIN", "SUM", "COUNT", "AVG",
     "VAR_POP", "VAR_SAMP", "STDDEV_POP", "STDDEV_SAMP", "COVAR_POP", "COVAR_SAMP", "CORR",
     "REGR_INTERCEPT", "REGR_R2", "REGR_SLOPE", "REGR_SXY")
-  private val supportedFunctions = supportedAggregateFunctions
+  private val supportedDatetimeFunctions = Set("DATE_TRUNC")
```

Comment:
`private val supportedAggregateFunctions: Set[String] = Set(
  "MAX", "MIN", "SUM", "COUNT", "AVG",
  "VAR_POP", "VAR_SAMP", "STDDEV_POP", "STDDEV_SAMP",
  "COVAR_POP", "COVAR_SAMP", "CORR", "REGR_INTERCEPT",
  "REGR_R2", "REGR_SLOPE", "REGR_SXY", "DATE_TRUNC"
)`

This approach simplifies the code and keeps everything in one place. Let me know what you think!

Reply:
Imo DATE_TRUNC should not be in the supportedAggregateFunctions as it's not an aggregate function